### PR TITLE
New about

### PR DESCRIPTION
**Pull request by lsywua3**
弄了新的about + theme + form的部分。（可以中文切换，浏览器缩小的时候页面也会有调整了）
event部正在商讨Theme的三个logo下面是否还需要放一些文字信息。
theme和form里使用的logo的网站要求放上logo来源的链接，我现在放在theme和form的左下角了，到时候也许可以一起放到footer上。
这3个部分里使用到的颜色如果需要调整的话就跟我说~

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -78,45 +78,107 @@
...
+    <div class="theme-header">
+    <div class = "border-2"></div>
+    <h1 class = "lang" data-key="theme-header">2020 Themes</h1>
+    <div class = "border-3"></div>
+    </div>
+    <div class = "theme-demo">
+        <div class="first-theme">
+            <img id = "ai-icon" src="https://img.icons8.com/cotton/100/000000/artificial-intelligence.png"/>
+            <h2 class="lang" data-key="theme-1-1">Artificial Intelligence</h2>
+ <!--           <p>One important theme for this event is Articial Intelligence. Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. <span id = "more-1">It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.</span></p>
```
Comment (Falanke21) [inline index.html:114]: Are these comments suppose to be added up later? If not, we don't need them here.

Comment (Falanke21) [reply index.html:114]: If so, we can leave it here. I think you never used expandText()?

**Review by Falanke21 — CHANGES_REQUESTED** [review]
image

I notice a bug here: the icon source link, shouldn't be clicked when the cursor is to the right of the link.

**Comment by Falanke21** [comment]
Otherwise, this PR looks astonishing. Great design!

**Comment by lsywua3** [comment]
I fixed the icon bug~ 
For the commentted code mentioned above, I want to keep it because the event department might choose to add some theme related text information under the theme section, so if they eventually decide to add texts, I will add the text directly using that commented code.
The expandText() is made to avoid the theme text being long because it might easily look like there are too many texts under that section if the texts are a bit long, so I want to keep this function as well for now : )

**Pull request merged**